Make the Certificate and Roles tables design as the rejection reason table

ilja903 opened the pull request:
https://unctad.atlassian.net/browse/EV2CORE-66

kamsi commented:
@ilja903 fix the conflicts, please.

ilja903 commented:
resoved

kamsi commented on view/statistics-flow-certificates.js:
```diff
@@ -140,10 +140,10 @@ exports['statistics-main'] = function () {
-				var mode = modes.get(location.query.mode || 'daily');
-				return table({ class: 'statistics-table' },
+			var mode = modes.get(location.query.mode || 'daily');
+			return div({ class: 'overflow-x width-1050px' },
```
I'm not sure that those classes are needed. However, what is needed is `responsive-container` class (See rejection reasons table for reference).

kamsi commented on view/statistics-flow-roles.js:
```diff
@@ -173,10 +173,10 @@ exports['statistics-main'] = function () {
-				var mode = modes.get(location.query.mode || 'daily');
-				return table({ class: 'statistics-table' },
+			var mode = modes.get(location.query.mode || 'daily');
+			return div({ class: 'overflow-x width-1050px' },
```
responsive container may not work here... still that should not happen:

![layout_problem](https://cloud.githubusercontent.com/assets/7392217/25737598/045f7c6e-3179-11e7-8dca-0d5315eb92f6.png)

ilja903 commented:
Great code review, width class was totally useless, I removed it. Also fixed the mobile version. However put the comment to Jira next time, Vianney does not read github.
